shared/actions/git.js

```diff
@@ -95,12 +96,15 @@ const clearBadgesAfterNav = (state, action) => {
   } else if (_wasOnGitTab) {
     _wasOnGitTab = false
     // clear badges
-    return RPCTypes.gregorDismissCategoryRpcPromise({
-      category: 'new_git_repo',
-    }).catch(logError)
+    return clearNavBadges()
   }
 }
 
+const clearNavBadges = () =>
```

Comment:
we now clear badges on clicking the tab, but keep the new items orange until you unmount